frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -76,13 +76,16 @@ used with multiple versions of OpenShift Console but don't provide any backwards
 
 ## OpenShift Console Versions vs SDK Versions
 
-Not all NPM packages are fully compatible with all versions of the Console. This table will help align
-compatible versions of distributable SDK packages to versions of the OpenShift Console.
+Console plugin SDK packages follow a semver scheme where the major and minor version number indicates
+the supported OCP Console version, and the patch version number indicates the release of that particular
+package. For example, package version `4.19.0` is the initial package release which supports Console 4.19.
+
+For older 1.x plugin SDK packages, refer to the following version compatibility table:
 
 | Console Version | SDK Package                                     | Last Package Version |
 | --------------- | ----------------------------------------------- | -------------------- |
-| 4.17.x          | `@openshift-console/dynamic-plugin-sdk`         | Latest               |
-|                 | `@openshift-console/dynamic-plugin-sdk-webpack` | Latest               |
+| 4.17.x          | `@openshift-console/dynamic-plugin-sdk`         | 1.8.0                |
```

Comment:
This table needs an update, latest Console 4.18.x plugin SDK packages are following:
- `@openshift-console/dynamic-plugin-sdk` => 1.8.0
- `@openshift-console/dynamic-plugin-sdk-webpack` => 1.3.0

I need to trace back the package versions for Console 4.17.x

Reply:
Should we publish 4.18.0 SDK packages when we GA 4.18?

Reply:
@spadgett Yes, we can re-publish them under new version scheme e.g. `4.18.0` when Console 4.18 becomes GA.